lib/stack-driver-transform.js

```diff
@@ -20,6 +20,10 @@ function processUser({ authorized, authorizedModel }) {
         return null;
     }
 
+    if (typeof authorizedModel === 'object' && authorizedModel.modelName) {
```

Comment:
I'm beginning to think it might be worth adding the ability to pass in a custom message formatter which can be injected here. That could well be over-engineering though, so this is almost certainly fine for now.